src/features/MoviesBrowser/moviesBrowserAPI.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+// const APIkey = "https://api.themoviedb.org/3/movie/11?api_key=f577b54446e4b39433952b4514de7d17";
```

Comment:
I think you can delete commented part if don't use

Reply:
Thanks, I must have missed that